src/test/java/com/notably/commons/compiler/tokenizer/TokenizerTest.java

```diff
@@ -0,0 +1,34 @@
+package com.notably.commons.compiler.tokenizer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class TokenizerTest {
+    @Test
+    public void tokenize() {
```

Comment:
Also, as per suggested naming convention, can consider renaming this test to "testTokenize".
https://nus-cs2103-ay1920s2.github.io/website/book/cppToJava/junit/basic/

Reply:
@HemanshuGandhi Hmm I don't think that's the correct naming convention. Take a look at [this](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html)!